### DISCUSSION TITLE
Pushover Alert Error Notification

### DISCUSSION THREAD

**Discussion by steffanvigano** [juftin/camply#350] [General]
Just in case you haven't heard/see this yet.  I got the following error recently:

camply encountered an error and exited 😟 [2024-05-27 21:07:32] - (ConnectionError) {"token":"invalid","errors":["application has exceeded its monthly limit of 10000 messages, see https://support.pushover.net/i13"],"status":0,"request":"bac3fc09-c864-4c13-a5b4-eefa01cca8f5"}

Thanks

**Comment by juftin** [comment]
Hey @steffanvigano there's a fix out for this already - see https://github.com/juftin/camply/issues/348

**Reply by juftin** [reply]
Sadly this has happened once before where more than 10K pushover messages were sent out (the culprit last time was me) - I made a change that should hopefully prevent this but I updated the documentation to recommend everyone creates their own Pushover App token https://juftin.com/camply/command_line_usage/#send-a-push-notification

Thanks for posting the heads up on this and helping keep camply running for everyone!

**Comment by steffanvigano** [comment]
Thanks for the background info.  It wasn't clear at first if it was my token or what.  I was thinking there was no way I crested over.  Of note, I get the updates from GitHub when changes are made, but often I can't seem to correlate those to any specific bug/topic.  Probably just me.  It would be nice if there was a way to reference bug fixes to discussion topics.  Cheers.

**Comment by juftin** [comment]
Yeah, assuming I reference the issues / PR correctly in my commits this is handled in the release notes automatically. Here's an example where I did it right: https://github.com/juftin/camply/releases/tag/v0.32.6

[Last night's release](https://github.com/juftin/camply/releases/tag/v0.32.7) didn't reference the issue though, I'll be better about that in the future.
